curriculum/challenges/english/25-front-end-development/lecture-what-is-css/672acc03c257524d6a5151e8.md

```diff
@@ -98,15 +98,15 @@ CSS allows various units to set dimensions.
 
 ---
 
-`px`, `vh`, and `vw`.
+`pxx`, `depth`, and `radius`.
 
 ### --feedback--
 
 CSS allows various units to set dimensions.
```

Comment:
this feedback doesn't explain why the answer is wrong nor hints to the right one, maybe it needs some expanding?

Reply:
done!